rtf

```diff
@@ -322,6 +323,16 @@ class Host:
       shutil.copy(src, key_directory+'/'+self.hostid)
     return output
 
+  def get_version(self):
+    """Get rudder version (x.y) if any"""
+    if 'rudder-version' not in self.info:
+      return None
+    m = re.search('(\d+\.\d+)', self.info['rudder-version'])
+    if m:
+      return float(m.group(1))
```

Comment:
I don't think we should use floats for versions

Reply:
yes only major+minor i hope that is more than enough for a generic installation procedure